modules/standard/IO.chpl

```diff
@@ -8694,13 +8732,23 @@ proc _channel.skipField() throws {
   :arg args: the arguments to format
   :returns: the resulting string
 
-  :throws SystemError: Thrown if the string could not be formatted.
+  :throws EofError
+  :throws UnexpectedEofError
+  :throws BadFormatError
+  :throws SystemError: Thrown if the string could not be formatted for a less
```

Comment:
To keep the wording consistent across the module, I'd reuse: "... could not be formatted *for another reason*" here.